src/modules/direction/store/directionSlice.ts

```diff
@@ -275,6 +276,7 @@ export const fetchMaterials = (direction: IDirection): AppThunkType => async (
         page: meta.pageNumber + 1,
         size: LOAD_POSTS_LIMIT,
         type: postTypes,
+        tag: postTags,
```

Comment:
please make sure it works fine if these values are undefined

Reply:
we have an empty array by default and when added value to postTypes and postTags I used question mark so that undefined cannot be added. I checked it with wrong request and got empty array.